app/models/concerns/completed_run.rb

```diff
@@ -67,7 +67,7 @@ def total_playtime
       time = 0
       return time unless program == "livesplit"
       segments_with_history.each do |segment|
-        time += segment.history.sum { |h| h[:duration_seconds].nil? ? 0 : h[:duration_seconds] }
+        time += segment.history.sum { |h| h[:duration_seconds].blank? ? 0 : h[:duration_seconds] } || 0
```

Comment:
Fix seems logical, but this line is getting to be a mouthful. Can we make this more readable somehow?